Felienne/hedy
# [BUG FIX] Fixes learn-more page UI

**Description**
In this PR we fix the learn more page UI by only storing the subscribe newsletter form inside the white shadow box instead of all content. This should have already been the case, but a missing closing `div` tag was the boosdoener.

**Fixes**
This PR fixes #3065.

**How to test**
This is only a visual change. Navigate to the learn-more page and verify that only the subscribe form is inside a white shadowbox.
